src/drafts/SelectPanel2/SelectPanel.tsx

```diff
@@ -377,12 +377,7 @@ const SelectPanelSearchInput: React.FC<TextInputProps> = ({onChange: propsOnChan
           }}
         />
       }
-      sx={
-        {
-          /* TODO: uncommenting this breaks keyboard navigation, that's odd */
-          // '& input:empty + .TextInput-action': {display: 'none'},
-        }
-      }
+      sx={{'&:has(input:placeholder-shown) .TextInput-action': {display: 'none'}}}
```

Comment:
Note: this was fun! was able to use a [new css feature](https://caniuse.com/css-has) and a [really old css feature](https://caniuse.com/css-placeholder-shown)